arango/collection.py

```diff
@@ -1385,8 +1385,8 @@ def add_geo_index(
             with at least two floats. Documents with missing fields or invalid
             values are excluded.
         :type fields: str | [str]
-        :param ordered: Whether the order is longitude, then latitude.
-        :type ordered: bool | None
+        :param geo_json: Whether to use GeoJSON data-format or not.
```

Comment:
Might be helpful to mention the change in the docs, so that users may spot the issue easily.
```
Whether to use GeoJSON data-format or not. This parameter has been renamed from `ordered`.
```

Reply:
good call, added in [5b0d0ae](https://github.com/ArangoDB-Community/python-arango/pull/318/commits/5b0d0aec9555231c5a8428fa6cac5fbc69209d88)